.github/workflows/tests.yaml

```diff
@@ -360,6 +360,8 @@ jobs:
             path: tests/escrow
           - cmd: cd tests/pyth && anchor test --skip-lint && npx tsc --noEmit
             path: tests/pyth
+          - cmd: cd tests/realloc && anchor test --skip-lint && npx tsc --noEmit
```

Comment:
A simple thing, but it might be better to not crease another workflow, but add it into existing (`etc`) one,